app/services/animal_service.rb

```diff
@@ -0,0 +1,15 @@
+class AnimalService
+  def conn
+    Faraday.new(url: "http://localhost:3000") do |faraday|
+    end
+  end
+
+  def get_url(url)
+    response = conn.get(url)
+    JSON.parse(response.body, symbolize_names: true)
+  end
+
+  def get_animal(animal_id)
+    get_url("api/v1/animals/#{animal_id}")
```

Comment:
we'll have to get together to talk about how these requests are gonna look. like if we should include the user/shelters in the animal request. I honestly don't know which one would be better or if it matters

Reply:
Good point! I did not think about that.